src/client/components/Forms/ChangePasswordForm.test.tsx

```diff
@@ -0,0 +1,39 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+import { ChangePasswordForm } from "@coin-view/client";
+import { I18nextProvider } from "react-i18next";
+import { instanceEN } from "@coin-view/mocks";
+
+const onSubmit = jest.fn();
+
+describe("Change Password Form", () => {
+  it("Basic render", () => {
+    render(
+      <I18nextProvider i18n={instanceEN}>
+        <ChangePasswordForm onSubmit={onSubmit} />
+      </I18nextProvider>
```

Comment:
For occurring usage of provider declare a `Wrapper`.
For example
```tsx
const Wrapper = ({children}) => {
   <I18nextProvider i18n={instanceEN}></I18nextProvider>
}
```

Then just use: 

```tsx
<Wrapper>
   <ChangePasswordForm onSubmit={onSubmit} />
</Wrapper>
```

It's even more useful when there is more than one Provider.
You can use `Providers` name instead of `Wrapper`. Your choice.
This applies to all the other tests in this pr.